anomaly_detector/anomaly_detector.py

```diff
@@ -215,11 +222,28 @@ def infer(self):
         # When we reached # of inference loops, retrain models
         self.recreate_models = False
 
+    def fetch_false_positives(self):
+        """Fetch false positive from datastore and add noise to training run."""
+        _LOGGER.info("Fetching false positives from fact store")
+        try:
+            r = requests.get(url=self.config.FACT_STORE_URL + "/api/false_positive")
+            data = r.json()
+            data["feedback"]
+            fp = []
+            for msg in data["feedback"]:
+                noise = [{"message": msg}] * self.config.FREQ_NOISE
+                fp.extend(noise)
+            _LOGGER.info("Added noise {} messages ".format(len(fp)))
+            return fp
+        except Exception as ex:
+            _LOGGER.debug("Fact Store is either down or not functioning")
```

Comment:
Should the used log level for the message ber `error`? There is an expection after all and you are reporting it as `debug` here which would not ever get printed in prod environment

Reply:
Woops. Thanks... Good catch.

Reply:
Fixed